.gitignore

```diff
@@ -25,3 +25,4 @@ dist
 yarn.lock
 .env
 .idea
+__snapshots__/
```

Comment:
@pbkabali why are we ignoring snapshots?

Reply:
@collinewait, I followed this advice since the snapshot gets obsolete as soon as there is any change on the tested file. This means we shall have many obsolete snapshots committed to version control since we are still making many changes to our files. At the moment we want to avoid that behavior

Reply:
You are right but we can still update snapshots with a `-u` flag when running tests to cater for the changes.

Reply:
Resolve conflicts too.

Reply:
Yes, you are very right. Thanks for that. I think we will stick to recommended practices and commit them. Let me work on that and conflicts too.

Reply:
This has been worked on.